### PR TITLE
Us059 remove loaded ci

### PR DESCRIPTION
**Pull request by tej99**
As a: CEC
I need: to be able to remove a loaded SEFT CI
So that: the correct CI is associated to the RU

https://trello.com/c/LhzrsVoU/69-us059-remove-seft-loaded-ci-s

### CODE REVIEW

**Comment by tej99** [comment]
[US059: Remove SEFT Loaded CI (s)](https://trello.com/c/LhzrsVoU/69-us059-remove-seft-loaded-ci-s)

File: response_operations_ui/controllers/collection_instrument_controllers.py [hunk 1]
```diff
@@ -44,3 +44,16 @@ def link_collection_instrument(ce_id, ci_id):
 
     logger.debug('Successfully linked collection instrument to collection exercise', ce_id=ce_id, ci_id=ci_id)
     return True
+
+
+def unlink_collection_instrument(ce_id, ci_id):
+    logger.debug('Unlinking collection instrument and collection exercise', ce_id=ce_id, ci_id=ci_id)
```
Comment (andrewmil) [inline response_operations_ui/controllers/collection_instrument_controllers.py]: In the logs we should probably be explicit, `collection_exercise_id=ce_id` etc

File: response_operations_ui/templates/collection-exercise.html [hunk 2]
```diff
@@ -12,6 +12,13 @@ <h1 class="jupiter" name="page-ce-title">{{survey.shortName}} {{ce.exerciseRef}}
         </div>
       </div>
     {% endif %}
+    {% if ci_unlinked %}
```
Comment (andrewmil) [inline response_operations_ui/templates/collection-exercise.html]: This all works fine, It maybe would be nice to eventually generalise the UI for these information panels so we don't just keep adding if statements

Comment (tej99) [reply response_operations_ui/templates/collection-exercise.html]: Yes definitely, i'll try put up another PR for this.

Comment (tej99) [reply response_operations_ui/templates/collection-exercise.html]: https://github.com/ONSdigital/response-operations-ui/pull/136

**Pull request merged**